postgres/datadog_checks/postgres/statement_samples.py

```diff
@@ -95,6 +111,13 @@ def __init__(self, check, config):
     def cancel(self):
         self._cancel_event.set()
 
+    def _dbtags(self, db):
```

Comment:
Let's support `*extra_tags`

Reply:
What are you referring to? I'm not sure what this means.

Reply:
```python
    def _dbtags(self, db, *extra_tags):
        tags = ['db:{}'.format(db)]
        tags.extend(extra_tags)

        if self._tags_no_db:
            tags.extend(self._tags_no_db)

        return tags
```

then change occurrences to use it

Reply:
Oh I see what you mean, updated https://github.com/DataDog/integrations-core/pull/9252/commits/ea12f6b1d615885eaf6851ee6f48659b0a419b42